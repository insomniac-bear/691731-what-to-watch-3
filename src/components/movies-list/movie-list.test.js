import React from 'react';
import renderer from 'react-test-renderer';
import MovieList from './movies-list.jsx';

const films = [
  {
    id: 0,
    filmName: `Name-0`,
    posterUrl: `img/bohemian-rhapsody.jpg`,
    filmPreview: `Some film url`,
  },
  {
    id: 1,
    filmName: `Name-1`,
    posterUrl: `img/bohemian-rhapsody.jpg`,
    filmPreview: `Some film url`,
  },
  {
    id: 2,
    filmName: `Name-2`,
    posterUrl: `img/bohemian-rhapsody.jpg`,
    filmPreview: `Some film url`,
  },
  {
    id: 3,
    filmName: `Name-3`,
    posterUrl: `img/bohemian-rhapsody.jpg`,
    filmPreview: `Some film url`,
  },
  {
    id: 4,
    filmName: `Name-4`,
    posterUrl: `img/bohemian-rhapsody.jpg`,
    filmPreview: `Some film url`,
  },
];

it(`Render MovieList`, () => {
  const tree = renderer
    .create(<MovieList
      genre={`All`}
      films={films}
      cardClickHandler={() => {}}
    />, {
      createNodeMock: () => {
        return {};
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
