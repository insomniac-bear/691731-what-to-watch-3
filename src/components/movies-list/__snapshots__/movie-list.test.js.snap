// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieList 1`] = `
<div
  className="catalog__movies-list"
/>
`;
